spec/writers/sparql_statement_writer_spec.rb

```diff
@@ -0,0 +1,190 @@
+# frozen_string_literal: true
+
+require 'rialto/etl/writers/sparql_statement_writer'
+require 'rialto/etl/readers/sparql_statement_reader'
+require 'rdf'
+require 'sparql'
+
+RSpec.describe Rialto::Etl::Writers::SparqlStatementWriter do
+  subject(:writer) { described_class.new({}) }
```

Comment:
Is the empty hash argument here needed? If so, :ok:. If not, can drop `({})` altogether.

Reply:
`{}` is an empty config, which is a required arg.

Reply:
> `{}` is an empty config, which is a required arg.

Ah, that's from `Traject::LineWriter`—good catch!